README.md

```diff
@@ -65,8 +65,10 @@ The `src/ragger/backend/interface.py` file describes the methods that can be imp
 * `right_click`: perform a right click on a device.
 * `left_click`: perform a left click on a device.
 * `both_click`: perform a click on both buttons (left + right) of a device.
+* `navigate`: performs a click on buttons of the device depending of the navigation instruction.
```

Comment:
I would reword it a bit : 

`navigate` : navigate on the device according to a set of navigation instructions provided.
`navigate and compare` : same as the previous method but take screenshots at every step and compare them with "golden images" after navigation is performed.